stdlib/Markdown/src/Markdown.jl

```diff
@@ -8,6 +8,9 @@ module Markdown
 import Base: show, ==, with_output_color
 using Base64: stringmime
 
+# Margin for printing in terminal.
+const margin = 2
```

Comment:
Could try juggling the `include`s around instead of moving this here. I would try moving 

```julia
include("GitHub/GitHub.jl")
include("IPython/IPython.jl")
```

underneath 

```julia
include(joinpath("render", "terminal", "render.jl"))
```

It _seems_ that the code is organized in a way that GitHub and IPython are sort of full self-contained extensions with rendering and output.

Reply:
In fact, that was my first thought. However, I was afraid because then the dependency between `render.jl` and Github would not be very explicit. I understood the constant here as: this is the global configuration for printing in terminal. Every render that needs to print to terminal should use it. What do you prefer?

Reply:
@mortenpi I did some testing and I think it will be worse to do this.

If I move those two files, then I get the error `UndefVarError(:blocktex)` in `Julia/Julia.jl` because it is defined in `IPython/IPython.jl`. However, if I also move `Julia/Julia.jl`, then I get the error `UndefVarError(:LaTeX))` in `render/plain.jl` because it is defined in `IPython/IPython.jl`. Thus, if we choose this path, then, at best, the includes organization will be very messy (there are still a possibility to be impossible without any further modification).

Hence, I think that declaring this constant here is the best option here...

Reply:
> Hence, I think that declaring this constant here is the best option for now...

Yeah, I was hoping it would be as easy as just moving the includes. But since that breaks other stuff, then I agree that it's better to just move the constant.

Reply:
Moving it here seems ok (not to derail fixing this bug with wider structural problems).

In the future is should really be passed along with a rendering context of some kind. Throwing it into `IOContext` would be appropriate I suppose.